### DISCUSSION TITLE
DaCE on novel AI Accelerators

### DISCUSSION THREAD

**Discussion by Sameeranjoshi** [spcl/dace#1436] [Ideas]
I was wondering if any work is done on exploring this framework on Novel AI Accelerators like Cerebras, Sambanova, Groq, etc... 
As these architectures are data flow so it might be interesting to give it a try. If any previous references or discussion on this topic are available and someone can point me that would be great. 

Thanks

**Comment by tbennun** [comment]
I think that the closest relatives to those accelerators are FPGAs, for which we have extensive support (including transformations to improve the dataflow aspects of applications). I would recommend to look at the relevant [documentation](https://spcldace.readthedocs.io/en/latest/optimization/fpga.html), [examples](https://github.com/spcl/dace/tree/master/samples/fpga), and [upcoming tutorial](https://github.com/spcl/dace/pull/1438).

**Comment by Sameeranjoshi** [comment]
Thanks I will start going through these links. In the meanwhile to get a better understanding of dataflow architectures, dataflow programming models and such are there any suggestions to start with some 101's I have zero knowledge of data flow models. Papers, book chapters or links which you found useful pointers to those would be great.
Thanks
